src/Symfony/Component/Cache/Adapter/AbstractTagAwareAdapter.php

```diff
@@ -37,14 +38,17 @@ abstract class AbstractTagAwareAdapter implements TagAwareAdapterInterface, TagA
 
     private const TAGS_PREFIX = "\0tags\0";
 
-    protected function __construct(string $namespace = '', int $defaultLifetime = 0)
+    private $marshaller;
+
+    protected function __construct(string $namespace = '', int $defaultLifetime = 0, MarshallerInterface $marshaller = null)
```

Comment:
Seems like marshaler can be made required argument here to simplify logic?
_1. this is ctor which does not follow BC rules (or?)
2. and even if it does, this is experimental in 4.3, so it's not stable before 4.4. is out_

Reply:
works for me, updated